train_gpt2.cu

```diff
@@ -1349,10 +1357,17 @@ int main(int argc, char *argv[]) {
     int recompute = 1; // recompute during backward setting, 0 = none, 1 = recompute gelu
     int zero_stage = 0; // Zero Optimization Stage for Multi-GPU training
     int hellaswag_eval = 0;
+    // multi-node settings
+    int num_processes = 1;  // this should be set by the slurm environment
+    int process_rank = 0;  // this should be set by the slurm environment
+    int gpus_per_node = 8;  // this should be set by the slurm environment
+    char nccl_init_method[256] = "mpi";  // "tcp" or "fs" or "mpi"
+    char server_ip[256] = "-1";  // used if init_method set to "tcp" -> set to your server ip address
+    char fs_path[256] = "/tmp";  // used if init_method set to "fs" -> set to a shared filesystem path
```

Comment:
wouldn't  /tmp usually be a path local to the current node?

Reply:
Kind of agree, in a previous PR it was `/dfs` or so, worried that a person might get a wrong impression of this flag if they see its default as `/tmp `.

Reply:
I'll change it to "" (empty) as it doesn't matter since the init method is set to "mpi" by default. This var is used only when init is set to "fs".